app/src/main/res/values-en/strings.xml

```diff
@@ -835,30 +835,30 @@ However, before uploading your changes, the app checks with a &lt;a href=\"https
     <string name="quest_cycleway_value_none">none</string>
     <string name="quest_cycleway_value_shared">lane shared explicitly with other traffic</string>
     <string name="quest_cycleway_value_suggestion_lane">bicycle suggestion lane</string>
     <string name="quest_cycleway_value_bus_lane">on bus lane</string>
-    <string name="quest_cycleway_value_separate">displayed separately on map</string>
+    <string name="quest_cycleway_value_separate">displayed separately on the map</string>
     <string name="quest_cycleway_value_sidewalk">explicitly shared sidewalk</string>
-    <string name="quest_cycleway_value_lane_dual">cycle lane, both directions</string>
-    <string name="quest_cycleway_value_track_dual">cycle track, both directions</string>
+    <string name="quest_cycleway_value_lane_dual">bike lane, both directions</string>
+    <string name="quest_cycleway_value_track_dual">separated bike path, both directions</string>
     <string name="quest_cycleway_value_none_but_no_oneway">none, but cyclists may use road in both directions</string>
     <string name="quest_cycleway_value_none_and_oneway">none, no cycling allowed in this direction</string>
-    <string name="quest_cycleway_answer_contraflow_cycleway">Also cycleway on other side…</string>
-    <string name="quest_cycleway_answer_no_bicycle_infrastructure">No cycleway at all</string>
+    <string name="quest_cycleway_answer_contraflow_cycleway">Also bike path on the other side…</string>
+    <string name="quest_cycleway_answer_no_bicycle_infrastructure">No bike lane or path at all</string>
     <string name="quest_cycleway_answer_no_bicycle_infrastructure_explanation">As usual, tap on one side of the illustration, select a proper answer and do the same for the other side.\n\nNote all the available options: some cases are not obvious, like a oneway road that allows cyclists in both directions!</string>
-    <string name="quest_cycleway_answer_no_bicycle_infrastructure_title">Selecting that there is no cycleway</string>
-    <string name="quest_cycleway_confirmation_oneway_for_cyclists_too">Are you sure that cyclists are not allowed to use this oneway street in both directions? In doubt, check if there is a sign at the end of the street.</string>
+    <string name="quest_cycleway_answer_no_bicycle_infrastructure_title">Selecting that there is no bike lane or path</string>
+    <string name="quest_cycleway_confirmation_oneway_for_cyclists_too">Are you sure that cyclists are not allowed to use this oneway street in both directions? If in doubt, check if there is a sign at the end of the street.</string>
 
-    <string name="quest_cyclewayPartSurface_title">What’s the surface of the cycleway here?</string>
+    <string name="quest_cyclewayPartSurface_title">What’s the surface of the bike path here?</string>
 
-    <string name="quest_cycleway_width_title">"What’s the width of this cycleway here?"</string>
```

Comment:
note that it includes asking also about contraflow, contraflow lanes, bike paths, cycling allowed on sidewalks etc